CHANGELOG/CHANGELOG-1.23.md

```diff
@@ -1198,6 +1198,24 @@ filename | sha512 hash
 - Update the system-validators library to v1.6.0 (#106323, @neolit123) [SIG Cluster Lifecycle and Node]
 - Upgrade etcd to 3.5.1 (#105706, @uthark) [SIG Cloud Provider, Cluster Lifecycle and Testing]
 - When using `RequestedToCapacityRatio` ScoringStrategy, empty shape will cause error. (#106169, @kerthcet) [SIG Scheduling]
+- This release enables in-tree RBD migration to CSI driver with a couple of feature gates.  These featuregates are alpha in this release.
+
+  - `CSIMigrationRBD`: when enable, it will redirect traffic from in tree rbd plugin ( kubernetes.io/rbd )  to CSI driver ( rbd.csi.ceph.com) , default to `false` now.
```

Comment:
```suggestion
  - `CSIMigrationRBD`: when enabled, it will redirect traffic from in tree rbd plugin ( kubernetes.io/rbd )  to CSI driver ( rbd.csi.ceph.com) , default to `false` now.
```